### ISSUE TITLE
CPU information wrong?

### ISSUE THREAD

**Issue by JamesClonk** [shipyard/shipyard#427]
I'm having trouble understanding the CPU setting for shipyard engines. Could anybody enlighten me please?

I've setup shipyard on a host and added a docker engine to it, with a CPU value of 1.0 (Because I assumed its for #amount of CPUs).
Then I started a container through the docker client with a cpu-shares setting of 512 (50%) like this:
```sh
$ docker run -d -P -c 512 -m 64m --name nginx nginx
```

But when I login to the shipyard web UI now and look at the containers, it shows me something like this:
```
Name 	CPUs 	Memory 	State
nginx 	5.12 	64 MB 	running 	
```

Why 5.12 CPUs? What does that mean?
The graphs on the main dashboard for CPU and the engine dashboard for CPU usage both also display a value of 5.12.
I was expecting for it be either 512 or 0.5 (or maybe 50%?), but not 5.12.
:confused:

The reserved cpus value from `shipyard info` looks similarly confusing:
```
$ shipyard info
Controller Version: 2.0.8
Cpus: 1.00
Memory: 416.00 MB
Containers: 1
Images: 15
Engines: 1
Reserved Cpus: 512.00% (5.12)
Reserved Memory: 12.50% (64.00 MB)
```

--- 

Docker version used:

```sh
$ docker info
...
Kernel Version: 3.13.0-40-generic
Operating System: Ubuntu 14.04.2 LTS
CPUs: 1
...
```
```sh
$ docker version
Client version: 1.5.0
Client API version: 1.17
Go version (client): go1.4.1
Git commit (client): a8a31ef
OS/Arch (client): linux/amd64
Server version: 1.5.0
Server API version: 1.17
Go version (server): go1.4.1
Git commit (server): a8a31ef
```

**Comment by ehazlett** [comment]
You set it as 1.0 CPU and then told the container to have 512. The CPU is
arbitrary and you schedule container based on that as a max. For example
0.1 for a container.

**Comment by JamesClonk** [comment]
Well, I started the container with `docker run` directly, not through the shipyard cli.
According to the documentation the cpu-shares flag is supposed to be a relative weight based on a value of 1024 being 100%. 
http://docs.docker.com/reference/run/#runtime-constraints-on-cpu-and-memory

It think that maybe shipyard/citadel has a bug in how it's supposed to calculate these CPU values for/from docker?

The calculation in citadel/engine.go:
```go
	config := &dockerclient.ContainerConfig{
		Hostname: i.Hostname,
		Domainname: i.Domainname,
		Image: i.Name,
		Cmd: i.Args,
		Memory: int(i.Memory) * 1024 * 1024,
		Env: env,
		CpuShares: int(i.Cpus * 100.0 / e.Cpus),
		ExposedPorts: make(map[string]struct{}),
	}
```

So this means, if I have an engine with 1.0 CPUs, and add a container (through the shipyard cli this time) with CPUs 0.5, it will calculate a CpuShares value of 50.
But if I go by the docker/cgroups documentation for cpu shares, a value of 50 out of 1024 is not 50%. It should be 512, as I did previously with `docker run -c 512`.
:question:
